package internal

import "errors"

var ErrTryAgain = errors.New("try again")
